enabler/src/payment-enabler/payment-enabler.ts

```diff
@@ -37,6 +37,16 @@ export interface PaymentEnabler {
   createComponentBuilder: (
     type: string
   ) => Promise<PaymentComponentBuilder | never>;
+
+  /**
+   * Creates a payment drop-in builder of the specified type.
+   * @param type - The type of the payment drop-in builder.
+   * @returns A promise that resolves to the payment drop-in builder.
+   * @throws {Error} If the payment drop-in builder cannot be created.
+   */
+  createDropinBuilder: (
```

Comment:
Please, notice that both methods `createComponentBuilder` and `createDropinBuilder` are mandatories. The idea is to force to implement both but do not do anything if one of them is not supported in a given connector.
The alternative is to make both optional. 

Please, let me know if this should be changed.